### ISSUE TITLE
This points structure is taking 20 MB in the compiled JS

### ISSUE THREAD

**Issue by valdo404** [raphoester/clickplanet.lol-frontend#13]
When the final artefact is produced, we can see that this structure is taking lots of place

https://github.com/raphoester/clickplanet.lol-frontend/blob/78a4e1e93986ffedf50cc3b66988f0e911e552df/src/app/viewer/points.ts#L15

Suggestion: putting it outside of the javascript file and load it using fetch + protobuf, or even better use ZIP.

Best

**Comment by raphoester** [comment]
Yep true was thinking about simply serializing in binary instead of json
